Explorer/Assets/DCL/Backpack/AuthenticationScreenCharacterPreviewController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DCL.Profiles;
+using System.Collections.Generic;
+
+namespace DCL.CharacterPreview
+{
+    public class AuthenticationScreenCharacterPreviewController : CharacterPreviewControllerBase
+    {
+        public AuthenticationScreenCharacterPreviewController(CharacterPreviewView view, ICharacterPreviewFactory previewFactory) : base(view, previewFactory)
+        {
+        }
+
+        public override void Initialize(Avatar avatar)
+        {
+            previewAvatarModel.Wearables = new List<string>();
+            foreach (var wearable in avatar.SharedWearables)
```

Comment:
Can we unify the type of wearable id in favor of `URN`? So we don't have such strange loops just to implicitly convert between types and in general it will help unification

Reply:
great point :D